service_thing_aggregate/test/controllers/ObjectControllerIntegrationSpec.scala

```diff
@@ -343,6 +343,22 @@ class ObjectControllerIntegrationSpec extends MusitSpecWithServerPerSuite {
         (objects.head \ "subNo").as[String] mustBe "34"
       }
     }
+
+    "finding object by uuid" should {
```

Comment:
event better wording `"searching for an object using UUID" should "successfully return the object" in ...`

A good rule of thumb is  that the test wording should read like a proper sentence.